pom.xml

```diff
@@ -1715,6 +1715,16 @@
                   <searchTransitive>true</searchTransitive>
                   <message>Banned log4j:log4j dependency/transitive dependency was found!</message>
                 </bannedDependencies>
+                <bannedDependencies>
+                  <excludes>
+                    <exclude>jline:jline</exclude>
+                  </excludes>
+                  <includes>
+                    <include>jline:jline:1.0</include>
```

Comment:
nit: please explain here with a brief comment why we still allow 1.0 to make it clear for the future if we can remove